scripts/deploy_new_chain.py

```diff
@@ -0,0 +1,25 @@
+from brownie import GasStationV2, accounts
+
+GITHUB_POKER="0x737760C760FfEc370F84861E4Be4AFF7093Ffa3f"
+OMNICHAIN_MULTISIG="0x9ff471f9f98f42e5151c7855fd1b5aa906b1af7e"
+def deploy_chain(account, chain):
+    keeper = OMNICHAIN_MULTISIG
+    GasStationV2.deploy(keeper, 60*60*8, {"from": account})
+    gs = GasStationV2[len(GasStationV2)-1]
+    GasStationV2.publish_source(gs)
+    gs.transferOwnership(OMNICHAIN_MULTISIG)
```

Comment:
i would think ownership should be transferred last, or at least after the `addRecipients` calls?